### PR TITLE
Allow users to select between map with user input and bigfoot sightings

### PR DESCRIPTION
**Pull request by jiminbyun**
Users can now choose between two data to be displayed on the map using dropdown select menu. When clicked, it also shows appropriate descriptor message about each map so user can be more clear about how to interact with it. Also, initial center for each map has changed for better visualization: center of globe for user input and center of US for bigfoot sightings. 

Screenshot 2020-06-18 at 11 33 37 AM - Display 1

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -188,26 +188,33 @@ function onload() {
...
+  const messageContainer = document.getElementById("map-descriptor");
+  messageContainer.innerText = "This is locations of Bigfoot sightings 🦧";
+
   const response = await fetch('/bigfoot-data');
   const bigfootSightings = await response.json();
 
   const map = new google.maps.Map(
     document.getElementById('map'),
-    { center: { lat: 35.78613674, lng: -119.4491591 }, zoom: 7 });
+    { center: { lat: 39.8283, lng: -98.5795 }, zoom: 4 });
```
Comment (darrnshn) [inline portfolio/src/main/webapp/script.js:217]: nit: if this PR is only about allowing users to switch between two maps, then maybe this belongs to a separate PR

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -286,3 +293,12 @@ function createWindowInput(latitude, longitude) {
 
   return infoWindow;
 }
+
+function selectMap() {
+  const option = document.getElementById("map-select").value;
+  if (option == "user") {
```
Comment (darrnshn) [inline portfolio/src/main/webapp/script.js:299]: nit: ===

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -83,7 +83,15 @@ <h3>Biorobotics Lab, Research Assistant, Spring 2019-Fall 2019</h3>
...
       <h2>Etc.</h2>
-      <div id ="map"></div>
+      <div id ="map-container">
+        <label for="map-select">Choose data to display:</label>
+        <select name="map-select" id="map-select" onchange="selectMap()">
+          <option value="user">Opinions from users</option>
+          <option value="bigfoot">Bigfoot Sightings</option>
+        </select>
+        <div id="map-descriptor"></div>
+        <div id ="map"></div>
```
Comment (darrnshn) [inline portfolio/src/main/webapp/index.html:93]: nit: `id="map"`

**Review by darrnshn — APPROVED** [review]

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (jiminbyun) [reply portfolio/src/main/webapp/script.js:217]: I'll keep this in mind for next PR!
